pkg/service/test/test.go

```diff
@@ -55,6 +55,20 @@ type TestOptions struct {
 	PassthroughHosts   []models.Filters
 }
 
+type TestReportVerdict struct {
+    	total int
+    	passed int
+    	failed int
+}
+
```

Comment:
since we've util file this struct can be moved there. Rest LGTM

Reply:
done